tests/unit/processes/file/test_cover_process.py

```diff
@@ -11,8 +11,11 @@ class TestCoverProcess:
     def testProcess(self, mocker):
         class TestCoverProcess(CoverProcess):
             def __init__(self, *args):
+                self.editions = set()
+                self.db_manager = mocker.MagicMock()
                 self.fileBucket = 'test_aws_bucket'
-                self.batchSize = 3
+                self.batch_size = 3
+                self.ingestLimit = None
```

Comment:
how do we decide if we should use snake case or camel case?

Reply:
That's a good question. We are in the midst of transitioning over to snake case which is a common practice. You will cases where there is a mix of both. I will double check my snake case here tho!